### PR TITLE
Edited all the titles of pages

### LINKED ISSUE #93: Some pages have the same title, so the title cannot be used to distinguish pages.
## Description
Some pages have the same title, so the title cannot be used to distinguish pages.
As a user, Title should be different for each page to avoid confusion.

## Mocks
All pages have same title.
screen shot 1939-12-07 at 5 55 26 pm
screen shot 1939-12-07 at 5 56 09 pm

## Acceptance Criteria
Each web page should have different titles so that the pages can be distinguished.
### Update [Required]
- [ ]  index.html
- [ ] code-of-conduct.html
- [ ] contactus.html
- [ ] member-levels.html
- [ ] newcomers.html
- [ ] open-source-workflow.html
- [ ] reporting-guidelines.html
- [ ] gsoc/2009.html
- [ ]  gsoc/2010.html
- [ ]  gsoc/2011.html
- [ ]  gsoc/2012.html
- [ ] gsoc/2014.html
- [ ] gsoc/2015.html
- [ ] gsoc/2016.html
- [ ]  gsoc/2017.html
- [ ] RGSoC/2018.html
- [ ] Outreachy/2017.html

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
15-20 min

### PR DESCRIPTION
**Pull request by Vertika-Shukla**
Edited the titles of all the pages so that they can be distinguished from each other. Solved Issue #93

### Description
Some pages had same page titles, this made it difficult to distinguish pages.
To avoid confusion, Edited all the titles of the pages for users. Relevant titles have been given to the pages.

Fixes #93 

### Type of Change:
- Code
  

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Code has been run on Browser to test it successfully

### Checklist:

- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: contactus.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
-    <title>Systers Open Source Projects by Systers</title>
+    <title>Contact Us! Systers</title>
```
Comment (sara-02) [inline contactus.html]: @Vertika-Shukla keeping it consistent with rest of titles we can have `Contact Us! -Systers` instead?

File: index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="chrome=1">
-    <title>Systers Open Source Projects by Systers</title>
+    <title>HOME- Systers Open Source Projects by Systers</title>
```
Comment (sara-02) [inline index.html]: @Vertika-Shukla again talking about consistence do we need a captialised `HOME`?
Camel Cased `Home` should do, right?

File: contactus.html (hunk 1 above)
Comment (Vertika-Shukla) [reply contactus.html]: @sara-02 Yup sure! Let me update it

**Comment by sara-02** [comment]
@Vertika-Shukla LGTM +1.
Please squash both your commits into a single commit, and your PR will be good to go :)

**Comment by Vertika-Shukla** [comment]
@sara-02 Done, I have made into one commit.

**Comment by BaaniLeen** [comment]
@Vertika-Shukla  Please resolve the merge conflicts for this PR to be approved. The names of the folders Outreachy & RGSoC have been changed to outreachy & rgsoc respectively.

**Comment by divyanshu-rawat** [comment]
@Vertika-Shukla Yes, Please resolve the merge conflict.

**Comment by ancyp** [comment]
@divyanshu-rawat close?

**Comment by divyanshu-rawat** [comment]
yes @ancyp sure go on.

**Pull request closed without merging**